### ISSUE TITLE
Follow-up with Cell Model Passports

### ISSUE THREAD

**Issue by Afollet** [PDXFinder/pdxfinder#755]
Initial contact has been made with Cell Model Passports and the PDX Finder requirements declared. Therefore, the PDX Finder needs to follow-up to attain the following information:

* A list of models with their model validation
* Logo and source description
* contact email and name
* mutation data, once it was updated to the 38 assembly and corresponding platform information. This can be de-prioritized for the release if necessary

##### ACCEPTANCE CRITERIA #####
- [x] Correspondance at CMP is contacted
- [x] information is integrated and checked
- [ ] molecular data is loaded if possible

**Comment by Afollet** [comment]
I'm still waiting on correspondence with CMP.

**Comment by Afollet** [comment]
I'm sending a follow-up email today. I think a separate ticket should be made for the molecular data.

@CsabaHalmagyi Done
